docs/how-do-i-install-my-operator-with-olm.md

```diff
@@ -1,6 +1,17 @@
-# How do I install my operator with OLM? 
+# How do I install my operator with OLM?
+
+[Once you've made your packaged operator available in a catalog](packaging-an-operator.md) it will appear in the `packagemanifest` list from which the Operators to install are selected. See [How do I list available Operators](list-available-operators.md#information-relevant-for-installation) how to retrieve the required information from the `PackageServer` in order start an installation of an Operator.
+
+## Install with automatic updates enabled
+
+First, retrieve the package name, channel and catalog name and catalog source namespace from the `packagemanifest` of the desired Operator to install.
```

Comment:
nit: package name, channel, `CatalogSource` name, and `CatalogSource` namespace from the `packagemanifest`...